### PR TITLE
tests: subsys: suit: add suit tag for samples

### PR DESCRIPTION
**Pull request by nordic-piks**
Extend list of paths used to check if suit
samples should be executed.

### CODE REVIEW

**Comment by nordic-piks** [comment]
The idea behind this change is to allow automatic detection of PRs not related to given code area.
In this case, there is defined tag (e.g. suit) and files/folder with source code related to that area.
This list of files/folders is then compared to files changed in PR. If particular PR does not change them,
tests related to specified tag should not be executed - will be excluded.

**Review by tomchy — APPROVED** [review]

File: scripts/ci/tags.yaml [hunk 1]
```diff
@@ -97,7 +97,14 @@ posix:
   files:
     - zephyr/lib/posix/
 
-# cbprintf:
-#    files:
-#        - zephyr/lib/os/cbprintf*
-#        - zephyr/lib/posix
+suit:
```
Comment (robertstypa) [inline scripts/ci/tags.yaml:100]: test-sdk-dfu is triggered by changes done in the following files/folders (test-spec.yaml):
```
"CI-suit-dfu-test":
  - "include/bl*"
  - "include/fprotect.h"
  - "include/dfu/*suit*"
  - "include/sdfw/sdfw_services/*"
  - "cmake/**/*"
  - "sysbuild/**/*"
  - "lib/dk_buttons_and_leds/**/*"
  - "subsys/dfu/*"
  - "subsys/nrf_rpc/**/*"
  - "subsys/nrf_security/**/*"
  - "subsys/sdfw_services/**/*"
  - "subsys/suit/**/*"
  - "samples/suit/**/*"
  - "subsys/mgmt/suitfu/**/*"
  - "tests/subsys/suit/**/*"

"CI-suit-dfu-test-extended":
  - "include/dfu/*suit*"
  - "include/sdfw/sdfw_services/*"
  - "subsys/sdfw_services/**/*"
  - "subsys/suit/**/*"
  - "samples/suit/**/*"
  - "subsys/mgmt/suitfu/**/*"
  - "tests/subsys/suit/**/*"
```
List above has been created during dependency analysis done by the team responsible for SUIT.
In my opinion the same files/folders shall be used to limit Twister tests/builds - if there is a plan to do so.

**Review by robertstypa — CHANGES_REQUESTED** [review]

File: scripts/ci/tags.yaml (hunk 1 above)
Comment (nordic-piks) [reply scripts/ci/tags.yaml:100]: Updated after evaluation of:
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml#L521-L545
https://github.com/nrfconnect/suit-generator/blob/ad6502a937d1e6e11824d3fb546f3d2027c5766a/.github/test-spec.yml#L1-L2
https://github.com/nrfconnect/suit-processor/blob/05089f4f7452ff50f09e2e2d776105c630381b1e/.github/test-spec.yml#L1-L2

Please note that the content is similar and syntax is different - the root path is NCS, all dependencies - from west modules - are stated at one place, tests and samples should not be included here, they are handled separately (but they will be build if changed)

**Review by robertstypa — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Comment by nordic-piks** [comment]
Just rebased and added path for tests.

**Pull request merged**